### PR TITLE
added map for postcodes to get cssrs for deletion report

### PR DESCRIPTION
**Pull request by Jonopono123**
#### Work done
- Re-added establishmentsData map so we're not passing a null reference.

#### Tests
Does this PR include tests for the changes introduced?
- [ ] Yes
- [ ] No, I found it difficult to test
- [x] No, they are not required for this change

### CODE REVIEW

File: server/routes/reports/deleteReport/report.js [hunk 1]
```diff
@@ -65,7 +65,13 @@ const createTableHeader = (WS1) => {
...
 
 const addCSSRData = async (establishmentsData) => {
-  const results = await models.pcodedata.getLinkedCssrRecordsFromPostcode(establishmentsData.postcode);
+  const results = {};
+  await Promise.all(
+    establishmentsData.map(async (establishment) => {
+      models.pcodedata.getLinkedCssrRecordsFromPostcode(establishment.postcode);
+    }),
+  );
+  console.log(results);
```
Comment (sbt92) [inline server/routes/reports/deleteReport/report.js:74]: rm log

Comment (Jonopono123) [reply server/routes/reports/deleteReport/report.js]: Done

File: server/routes/reports/deleteReport/report.js [hunk 2]
```diff
@@ -65,7 +65,13 @@ const createTableHeader = (WS1) => {
 };
 
 const addCSSRData = async (establishmentsData) => {
-  const results = await models.pcodedata.getLinkedCssrRecordsFromPostcode(establishmentsData.postcode);
+  const results = {};
+  console.log(establishmentsData[0]);
```
Comment (sbt92) [inline server/routes/reports/deleteReport/report.js:69]: rm log

**Review by sbt92 — APPROVED** [review]
LGTM

**Pull request merged**